float64 timestamp
int16 dx
int16 dy
uint8 button
uint8 pressed